sdk/core/azure-core/src/main/java/com/azure/core/util/FluxUtil.java

```diff
@@ -103,6 +103,34 @@ private static void accept(ByteArrayOutputStream byteOutputStream, ByteBuffer by
             .flatMap(serviceCall);
     }
 
+    /**
+     * This method converts the incoming {@code subscriberContext} from {@link reactor.util.context.Context Reactor
+     * Context} to {@link Context Azure Context}, adds the specified tracing attributes and calls the given lambda
+     * function with this context and returns a single entity of type {@code T}
+     * <p>
+     * If the reactor context is empty, {@link Context#NONE} will be used to call the lambda function
+     * </p>
+     *
+     * @param serviceCall serviceCall The lambda function that makes the service call into which azure context
+     * will be passed
+     * @param contextAttributes The map of attributes to be set on the tracer spans.
+     * @param  <T> The type of response returned from the service call
+     * @return The response from service call
+     */
+    public static <T> Mono<T> withContext(Function<Context, Mono<T>> serviceCall,
```

Comment:
There's a bit of duplication of code with these `withContext` methods. The `withContext(Function)` overload should just call into this method with an empty map and you should be able to delete `toAzureContext()` method.

Reply:
Updated to call in the `withContext` overload with an empty map. However, `toAzureContext()` is still needed, for `fluxContext()` method.